scripts/aggregate-changelogs/config.yaml

```diff
@@ -92,11 +92,11 @@ categories:
     color: "#bfffb8"
   - name: Rest API
     color: "#919ea1"
-  - name: Tenant Cluster Releases for AWS
+  - name: Workload Cluster Releases for AWS
```

Comment:
is capitalisation funny here? would it be `Workload cluster releases for AWS`?

Reply:
Maybe, however that hasn't really changed in this PR and I don't want to include changes of many category names in this PR.

![image](https://user-images.githubusercontent.com/273727/103665962-507b7180-4f74-11eb-82eb-17652cab90b6.png)

Reply:
Sure, works for me